static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -341,7 +347,8 @@ CStudioAuthoring.Module.requireModule(
 
           const external = {
             acecode: '/studio/static-assets/js/tinymce-plugins/ace/plugin.min.js',
-            paste_cleanup: '/studio/static-assets/js/tinymce-plugins/paste_cleanup/plugin.js'
+            paste_cleanup: '/studio/static-assets/js/tinymce-plugins/paste_cleanup/plugin.js',
+            paste_text: '/studio/static-assets/js/tinymce-plugins/paste_text/plugin.js',
```

Comment:
Shouldn't we call this `past_from_word`? 

Is this specific to word or general paste clean up? If it works for anything, what's the difference to our `paste_cleanup` plugin? Could/should this one replace `paste_cleanup` all together?